RevenueCatUI/Helpers/ProcessedLocalizedConfiguration.swift

```diff
@@ -6,7 +6,9 @@ struct ProcessedLocalizedConfiguration: PaywallLocalizedConfiguration {
     var title: String
     var subtitle: String
     var callToAction: String
+    var callToActionWithIntroOffer: String
     var offerDetails: String
+    var offerDetailsWithIntroOffer: String
```

Comment:
maybe these should be optional, not all products have an intro offer

Reply:
Doing in the next PR 👍🏻